src/content/popup/Metadata/MetadataContainer.tsx

```diff
@@ -1,8 +1,11 @@
+import type { ReactElement } from 'preact/compat';
+import { useLayoutEffect, useState } from 'preact/hooks';
```

Comment:
Rather than use `useLayoutEffect` and `useState`, I wonder if we can use [Suspense/lazy](https://preactjs.com/guide/v10/switching-to-preact/#suspense) for this?

e.g. https://gist.github.com/birtles/00b13ab5889551d1ca818fd35cfd7e90

Alternatively, we could isolate the async lookup to the `EraInfoComponent` itself but I guess that would be a much more invasive change.

Reply:
On further thought, rather than using `Suspense`/`lazy` we could just make a simple component to handle the async request like so:

https://gist.github.com/birtles/655c79e5889964c9e0f77b2b11240a6a

It's not strictly correct because if you update the input `EraMeta` you could have a moment where stale timespan is shown but I'm pretty sure that will never happen and if it does we could work around it by setting a `key` on the component whose value is based on the input so that we destroy and recreate the component in that case.

Reply:
> Alternatively, we could isolate the async lookup to the `EraInfoComponent` itself but I guess that would be a much more invasive change.

I didn't really want to do that because I couldn't foresee what the consequences would be for the Cosmos fixture.

Reply:
> It's not strictly correct because if you update the input `EraMeta` you could have a moment where stale timespan is shown but I'm pretty sure that will never happen and if it does we could work around it by setting a `key` on the component whose value is based on the input so that we destroy and recreate the component in that case.

Since I'm new to React, I simply applied your diff and am fully trusting your expertise on this.